app/moves/middleware/set-filter.moves.js

```diff
@@ -8,10 +8,16 @@ function setfilterMoves(items = [], bodyKey) {
   return async function buildFilter(req, res, next) {
     const promises = items.map(item =>
       moveService
-        .getActive({
-          ...get(req, `body.${bodyKey}`, {}),
-          isAggregation: true,
-        })
+        .getActive(
```

Comment:
a) We're still passing the location ids through too - this means we'll still send both the location ids and the supplier id (not sure how that works at the backend) and also we'll still batch if the locations exceed the batch size
b) this [c|s]hould probably be better handled in `set-body.moves` (and similarly for the others).